doc/api/process.markdown

```diff
@@ -179,6 +179,55 @@ this, you can either attach a dummy `.catch(() => { })` handler to
 `resource.loaded`, preventing the `'unhandledRejection'` event from being
 emitted, or you can use the [`'rejectionHandled'`][] event.
 
+## Event: 'warn'
+
+Emitted whenever Node.js emits a "Process Warning". Process Warnings are
+similar to Errors in that they describe exceptional conditions that are being
```

Comment:
From context here it's not entirely unambiguous what 'Errors' are, exceptions or error events.